pwiz_tools/Skyline/Properties/Settings.settings

```diff
@@ -886,6 +886,21 @@
     </Setting>
     <Setting Name="RelativeAbundanceLogScale" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">True</Value>
-    </Setting>	
+    </Setting>
+    <Setting Name="SrmResultsSavedForPath" Type="System.String" Scope="User">
```

Comment:
I think it would be safe to rename "SrmResultsXxx" properties if we think a different naming scheme would be better ("ImportResultsXxx", or "OpenDataSourceXxx"?)
"SrmResultsDirectory" was never actually used in older versions of Skyline because it only applied if the document had never been saved, so we don't have to worry about backwards compatibility.
"SrmResultsSourceType" was the only SrmResultsXxx property that actually got used.